_data/students.yml

```diff
@@ -1,6 +1,15 @@
 # New members please add your credentials to a random position in the file.
 # This is to remove conflicts with other people adding their credentials.
 
+- name: Aditya Kumar
+  github: adityak1121
+  image: adityak1121-WebDev.png
+  twitter: adityak11212
+  facebook: Aditya Kumar
```

Comment:
You got your facebook username wrong please go to your profile and put the thing after fb.com/ here ita your username pls.